submissions/Roophee/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,428 @@
+            <article class="content__info-info">
+                <table class="content__info-info-table">
+                    <tbody><tr>
+                        <th class="table-header table-border">Возраст:</th>
+                        <td class="table-data table-border">16+</td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">Режисер:</th>
+                        <td class="table-data table-border"> Джон Ли Хэнкок</td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">Язык перевода:</th>
+                        <td class="table-data table-border">Русский</td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">Жанр:</th>
+                        <td class="table-data table-border">Боевик, Триллер, Приключение, Криминал</td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">Время:</th>
+                        <td class="table-data table-border">128 мин.</td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">Страна производитель:</th>
+                        <td class="table-data table-border">США</td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">Сценарий:</th>
+                        <td class="table-data table-border">
+                            Джон Ли Хэнкок
+                        </td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">В главных ролях:</th>
+                        <td class="table-data table-border">Дензел Вашингтон, Рами Малек, Джаред Лето, Кристофер Бауэр, Майкл Хайатт, Терри Кинни, Натали Моралес, Изабель Арраиса</td>
+                    </tr>
+                    <tr>
+                        <th class="table-header table-border">Описание:</th>
+                        <td class="table-data table-border">Заместителя шерифа округа Керн Джо Дикона по прозвищу Дики отправляют в Лос-Анджелес – обычная командировка для сбора улик. Но там его привлекают к поискам серийного убийцы, который терроризирует город. Охоту на маньяка возглавляет Джим Бэкстер, сержант департамента шерифа Лос-Анджелеса. Его впечатляет полицейское чутье Дики, и он неофициально привлекает его к расследованию. Они всё ближе к цели, но Бэкстер не догадывается, что тем самым они воскрешают призраков прошлого Дики, раскрывая такие тайны, которые могут всколыхнуть куда более темные воды, чем его нынешнее дело.</td>
+                    </tr>
+                </tbody></table>
+            </article>
+        </section>
+        <section class="content__video">
+            <div class="content__video-label">
+                <P>
+                    Смотреть трейлер
+                </P>
```

Comment:
Same as at line 47.

Reply:
Used button element and  p not needed anymore.